src/postman/operation.ts

```diff
@@ -1,16 +1,6 @@
-import {
-  HttpParamStyles,
-  HttpSecurityScheme,
```

Comment:
If these were unused before, how did tests pass / things get merged? Maybe we're missing something during CI?

Reply:
Linting is passing in CI

![image](https://user-images.githubusercontent.com/1416224/76590751-2f650400-64bc-11ea-90b5-c76ccf2bd2c0.png)


Maybe such rule has never been configured?